src/Root.tsx

```diff
@@ -48,6 +49,27 @@ export async function onLatestVersionChange(newVer: SemVer | null | undefined, o
 const Root = observer(() => {
   const { app, modals } = getRootStore();
 
+  React.useEffect(() => {
+    const searchParams = new URLSearchParams(document.location.search);
+    const result = searchParams.get("Diagram");
+    if (result === null) return;
+
+    // Replace URL-safe characters with base64 equivalents
+    let base64String = result.replace(/-/g, "+").replace(/_/g, "/");
+
+    // Add padding if necessary
+    while (base64String.length % 4 !== 0) {
```

Comment:
Can you tell me more about why this is needed?

Reply:
Removed, this is just for testing the decoding. Thank you for the reminder.